core/domain/exp_jobs_one_off_test.py

```diff
@@ -290,6 +292,136 @@ def test_first_published_time_of_exploration_that_is_unpublished(self):
             exp_first_published, exploration_rights.first_published_msec)
 
 
+class ExpSummariesContributorsOneOffJobTest(test_utils.GenericTestBase):
+
+    ONE_OFF_JOB_MANAGERS_FOR_TESTS = [
+        exp_jobs_one_off.ExpSummariesContributorsOneOffJob]
+
+    EXP_ID = 'exp_id'
+
+    USERNAME_A = 'usernamea'
+    USERNAME_B = 'usernameb'
+    EMAIL_A = 'emaila@example.com'
+    EMAIL_B = 'emailb@example.com'
+
+    def setUp(self):
+        super(ExpSummariesContributorsOneOffJobTest, self).setUp()
+
+    def test_contributors_for_valid_contribution(self):
+        """Test that if only one commit is made, that the contributor
+        list consists of that contributor's user id.
+        """
+        self.signup(self.EMAIL_A, self.USERNAME_A)
+        self.user_a_id = self.get_user_id_from_email(self.EMAIL_A)
+
+        exploration = self.save_new_valid_exploration(
+            self.EXP_ID, self.user_a_id)
+        job_id = exp_jobs_one_off.ExpSummariesContributorsOneOffJob.create_new()
+        exp_jobs_one_off.ExpSummariesContributorsOneOffJob.enqueue(job_id)
+        self.process_and_flush_pending_tasks()
+
+        exploration_summary = exp_services.compute_summary_of_exploration(
```

Comment:
compute_summary_of_exploration is the only function that makes the call exp_models.ExplorationRightsModel.get_by_id, and does some other stuff to what it retrieves to make it into an ExplorationSummary object. Although it's called compute, it's really just doing a get and modifying some stuff for readability/to play nicely with the ExplorationSummary class, as far as I can tell.